tests/regressions/block_matrix/matrix_hpx.hh

```diff
@@ -65,7 +65,8 @@ struct vector_t_client:
   typedef hpx::components::client_base<vector_t_client, vector_t_server>
```

Comment:
Unrelated to this PR, but should we just rename these files to have `cpp` and `hpp` extensions?

Reply:
Right, I can do a separate PR for this (see https://github.com/STEllAR-GROUP/hpx/issues/5014).

Reply:
Perfect, thank you!